completion/README.md

```diff
@@ -95,4 +95,15 @@ exec $SHELL -l
 
 ## Fish
 
-TODO (#315): installation instructions
+### Mac (via Homebrew)
+Please look at the section about [installation via Homebrew](../README.md#using-homebrew-macos)
+``brew install git-machete`` automatically installs fish completion files for ``git machete``.
+
+### Linux
+
+Place the completion script in `/path/to/fish/completions/` typically (`~/.config/fish/completions/git-machete.fish`). You may have to create this directory if it doesn't exist.
+
+```shell script
```

Comment:
Let's add a relevant `mkdir -p` to the script to make sure that the target directory exists first